src/exes/moc.ml

```diff
@@ -262,7 +269,7 @@ let () =
   Internal_error.setup_handler ();
   Arg.parse_expand argspec add_arg usage;
   if !mode = Default then mode := (if !args = [] then Interact else Compile);
-  Flags.compiled := (!mode = Compile || !mode = Idl);
+  Flags.compiled := !mode = Compile;
```

Comment:
Nice! One more reason we need to fix this.

Reply:
Unfortunately, removing the Pipeline.generate_idl broke moc.js so I just restored it...

Reply:
Two steps forwards, one step back I guess